active/Resources/StyleVersion2.md

```diff
@@ -0,0 +1,89 @@
+ThemeVersion API
+=
+
+# Background
+
+XAML (WinUI) has a resources feature where you can put objects in a "resource dictionary", then
+use the resources by referencing the key name. All of the styling for XAML controls
+is defined in a resource dictionary and referenced by the controls.
+But an application can override these by defining a resource dictionary with one or more matching keys.
+
+XAML ships such an overrides dictionary named
+[XamlControlsResources](http://msdn.microsoft.com/library/Microsoft.UI.Xaml.Controls.XamlControlsResources).
+This dictionary defines all the resources necessary for all of the built-in controls.
+Since the resources in this dictionary can change over time, a new `ThemeVersion` property is being added.
+
+> StylesVersion? ResourcesVersion?
+
+The default value of this property will be the latest version, but an app can set another value to force to a
+previous version.
+
+# API Pages
+
+## XamlControlsResources.ThemeVersion
+
+Set this property to specify which version of resources to use for the look and styling
+of XAML controls. Default value is Version2.

```

Comment:
How does this affect breaking change rules?

Reply:
The plan is that the default value will change from version to version.

Reply:
Maybe that's a prose problem in the spec, then. Something like:

> This value defaults to the Framework's preferred version, which at this time is `Version2`. Setting this property overrides the framework's preference, allowing your app to specify which theme version should be applied.

Reply:
Recommend: update to that, that makes sense.